docs/src/extend/custom-rules.md

```diff
@@ -267,13 +284,23 @@ module.exports = {
         };
     }
 };
+{% endraw %}
+```
+
+In the file to lint:
 
-// in the file to lint:
+```javascript
+// someFile.js
 
 var foo = 2;
 //  ^ error: Avoid using variables named 'foo'
+```
+
+In your tests:
+
+```javascript
+// avoid-name.test.js
 
-// In your tests:
 var rule = require("../../../lib/rules/my-rule");
```

Comment:
```suggestion
var rule = require("../../../lib/rules/avoid-name");
```

If we give this rule and its file a specific name, then we should update the RuleTester example to use that name instead of `my-rule`.

Also on line 308 (it seems to be too far from the changed lines so GitHub doesn't allow suggestions there)

```diff
-ruleTester.run("my-rule", rule, {
+ruleTester.run("avoid-name", rule, {
```